### PR TITLE
Relocate category nav

### LINKED ISSUE #5415: PNI - relocate category nav to below hero and update styling
- [x] Relocate category nav to below hero.
- [x] Update styling: center alignment, typography/spacing/states remain the same
- [x] Update contents: Categories (All, Best Of, Home Office), ordering and remove About the Guide

Note that we'll have separate tickets for implementing the sticky behaviour and mobile nav.

---
###  Design -> Dev Handoff Checklist

- [x] Design has been finalized
- [x] Link to related ticket: #5314
- [x] Link to Abstract (make sure it can be viewed by anyone)
- [x] Design changes been merged into the master Sketch file
  - Merged into master Sketch file via Abstract. (_Master > Buyers Guide.sketch > [Home Page Updates (Q4 2020](https://share.goabstract.com/07a9aca5-3903-4f00-828a-6ac8bebb5be3?sha=e49d77cc9da6a70e8c412cd985e28138a3091cd5))_).
- [x] Desktop and mobile mockups are included
  - [[Desktop] Homepage - Index](https://share.goabstract.com/e9fd07c9-605e-4ca9-b99c-7a3686de51cb?collectionLayerId=103e293c-16c0-48af-bac2-dc7d7545e5ea&mode=design) 
  - this issue is desktop only (mobile nav separate)
- [x] Name of the Sketch file (and specific page/section to look at, if applies)
- [x] Assets are included. Link: n/a
- [x] Hover state for elements are designed: Match what is currently live & make active category link black
PNI_secondary-nav-states-desktop

### PR DESCRIPTION
**Pull request by mmmavis**
Closes #5415

https://foundation-s-new-issue--zpn92s.herokuapp.com/privacynotincluded

**Code reviewer:**
I shifted some DOM nodes around and created a `hero` template block.

### CODE REVIEW

**Comment by mmmavis** [comment]
gonna update this PR and re-request for reviews

File: network-api/networkapi/buyersguide/templates/bg_base.html [hunk 1]
```diff
@@ -73,23 +73,21 @@
 
 {% block body_wrapped %}
 
-  <header>
```
Comment (mmmavis) [inline network-api/networkapi/buyersguide/templates/bg_base.html:76]: view without whitespace changes should make code review easier

**Review by sabrinang — CHANGES_REQUESTED** [review]
The selected, current category should be `#000000` to match the border-color underline

**Comment by mmmavis** [comment]
Updated!

**Review by sabrinang — APPROVED** [review]
Update looks good, thanks!

**Review by Pomax — APPROVED** [review]
lgtm!

**Comment by mmmavis** [comment]
yay!

**Pull request merged**